app/src/main/java/tech/ula/ui/SessionEditFragment.kt

```diff
@@ -189,6 +234,22 @@ class SessionEditFragment : Fragment() {
             return
         }
 
+        // If another session has the startOnBoot property, warn the user
+        if (checkbox_start_boot.isChecked) {
+            if (sessionPreferences.getStartOnBootSession() != session.name && sessionPreferences.getStartOnBootSession().isNotEmpty()) {
```

Comment:
Here's another candidate for using `session.id` in the conditional instead of `session.name`.